openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -192,27 +190,24 @@
 - control_key: IA-3 (3)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: partial
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        The system does not provide addresses to devices.  However, it does utilize a private IP space for 
+        communicable endpoints.  Within this space address lease parameters are irrelevant because the endpoints
+        themselves are ephemeral.  A single endpoint instance cannot obtain multiple addresses.
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        Addresses are tied to deployed communicable endpoints, which are audited.
+        
 - control_key: IA-3 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-320
+        This is an organizational control and is not applicable to this system.
```

Comment:
What capabilities of OpenShift support this process? 

For example, documentation around how OpenShift deploys an image, verifies it prior to placing into production, assigns an IP address, and then any future changes are done through redeployments (that iterate this process).